packages/python-numpy/build.sh

```diff
@@ -31,7 +32,7 @@ termux_step_configure() {
 	. ${_CROSSENV_PREFIX}/bin/activate
 	build-pip install wheel
 
-	LDFLAGS+=" -lpython${_PYTHON_VERSION} -lm"
+	LDFLAGS+=" -Wl,--no-as-needed -lpython${_PYTHON_VERSION} -l:libm.a"
```

Comment:
Is `--no-as-needed` needed here?

Reply:
Actually I don't think this flag is needed but missing this flag will make some `*.so`s not link against `python3.11` and some tests will fail...

Reply:
Remove this flag. When testing `LDFLAGS='-lpython3.11' python -c 'import numpy; numpy.test()'` is needed.